lib/content-services/src/lib/content-node-share/content-node-share.dialog.scss

```diff
@@ -66,4 +66,20 @@
     .mat-form-field-flex {
         align-items: center;
     }
+
+    @media screen and (max-width: 380px) {
+
+        .mat-dialog-actions {
+            margin-bottom: 0px;
+            padding: 2px 0;
+        }
+
+        .mat-dialog-content {
+            max-height: 52vh;
```

Comment:
why on max-width you change the height of the dialog? and why 52% of the viewport height?

Reply:
hi, i checked that on 400% zoom the close button was going out of viewport, so in order to adjust the contents at 400% zoom, i reduced the max-height of mat-dialog-content from 65vh to 52vh and now the close button is visibile after that as it gets more space to accomodate.

Reply:
I am using max-width in media screen because it is related to percentage in zoom, so when i make changes at equivalent of max-width it reflects in % in zoom.